test/unit/unit/parsing.test.ts

```diff
@@ -216,7 +217,7 @@ describe('parseGrammarTestCase', () => {
     let originalFile = fs
       .readFileSync('./test/resources/parser.test.dhall')
       .toString();
-    let crlfFile = originalFile.split('\n').join('\r\n');
+    let crlfFile = originalFile.replace(/\r?\n/g, '\n');
```

Comment:
certain git options will check out the file with windows line endings. This ensures that doesn't actually matter...we force the file to the line endings we want and then run the test.